scripts/examples/webos_example.sh

```diff
@@ -0,0 +1,42 @@
+#!/usr/bin/env bash
+#
+#    Copyright (c) 2022 Project CHIP Authors
+#
+#    Licensed under the Apache License, Version 2.0 (the "License");
+#    you may not use this file except in compliance with the License.
+#    You may obtain a copy of the License at
+#
+#        http://www.apache.org/licenses/LICENSE-2.0
+#
+#    Unless required by applicable law or agreed to in writing, software
+#    distributed under the License is distributed on an "AS IS" BASIS,
+#    WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+#    See the License for the specific language governing permissions and
+#    limitations under the License.
+#
+
+# Activating connectedhomeip build environment
+source scripts/activate.sh
+
+# Extract webOS SDK
+cat third_party/webos_sdk/v2.14.1/webos_sdk.tar* | (cd third_party/webos_sdk/v2.14.1/;tar xvzf -)
+
+# Grant execute permission for NDK install script
+chmod 555 third_party/webos_sdk/v2.14.1/webos-sdk-x86_64-cortexa7t2hf-neon-vfpv4-toolchain-2.14.1.g.sh
+
+# Install webOS OSE NDK
+echo "##### Install webOS OSE NDK #####"
+third_party/webos_sdk/v2.14.1/webos-sdk-x86_64-cortexa7t2hf-neon-vfpv4-toolchain-2.14.1.g.sh -d third_party/webos_sdk/v2.14.1 -y
```

Comment:
Sources are not directly clear in plain in your git , that fine i would suggest , here you pull an artifact and extract it

Reply:
@jmeg-sfy 
Yes. I tried to put all resources as a plain file, but failed due to its size.
I think I should have internal discussion with relevant team to reduce SDK size to be fit so that all resources could be stored as plain form in next PR.
Thank you for your review and understanding.